skrub/_datetime_encoder.py

```diff
@@ -45,35 +77,44 @@ class DatetimeEncoder(BaseEstimator, TransformerMixin):
     extract_until : {"year", "month", "day", "hour", "minute", "second",
         "microsecond", "nanosecond", None}, default="hour"
         Extract up to this granularity.
-        If all non-constant features have not been extracted,
-        add the "total_time" feature, which contains the time to epoch (in seconds).
         For instance, if you specify "day", only "year", "month", "day" and
-        "total_time" features will be created.
-        If None, only the "total_time" feature will be created.
+        features will be created.
+        If ``None``, no feature will be created.
+
     add_day_of_the_week : bool, default=False
         Add day of the week feature (if day is extracted).
```

Comment:
why can't we have the day of the week without the day number?

Reply:
If you mean why "`(if day is extracted)`", this is also outdated :)

Reply:
yes that's what I meant